magmartfront/src/APIs/deleteUser.ts

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+function DeleteUser( id: string ) {
+    return Promise.resolve(axios.delete(`http://localhost:3001/users/${id}/del`))
```

Comment:
Padronizar a chamada usando env
```suggestion
    return Promise.resolve(axios.delete(`${process.env.API_URL}/users/${id}/del`))
```